bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html

```diff
@@ -95,6 +99,20 @@ <h2>{{ _('When and How to Use These Guidelines') }}</h2>
   {% endtrans %}
   </p>
 
+{% if l10n_has_tag('participation_nov2018') %}
+  <p>
+  {% trans %}
+    While these guidelines / code of conduct are specifically aimed at Mozilla’s
+    work and community, we recognize that it is possible for actions taken outside
+    of Mozilla’s online or inperson spaces  to have a deep impact on community
```

Comment:
Nit: extra space after `spaces`. I know it's ignored in HTML, but might be nice to fix if these strings are to be extracted for translation?

Reply:
Copy-pasta... I'll clean it up.